.travis.yml

```diff
@@ -55,3 +79,24 @@ jobs:
         # Remove the dev depenedencies
         - npm prune
       script: npm run zip-and-upload
+    - stage: Update Documentation
+      if: tag
```

Comment:
It's unclear to me what this condition is doing.

Reply:
It evaluates if `TRAVIS_TAG=true` which means that it's a tag commit.

Reply:
So if we ever tag a commit for some other reason, it will cause this to be triggered? I'm fine with that, just curious.

Reply:
@imurchie That's correct yes. If we're ever not ready to merge the docs yet, then we can just close the PR that's created by the job.